packages/css-library/config.js

```diff
@@ -1,6 +1,25 @@
+const { transform } = require('@divriots/style-dictionary-to-figma');
+
 module.exports = {
   source: ['tokens/**/*.json'],
+  format: {
+    figmaTokensPlugin: ({ dictionary }) => {
+      // currently Figma only supports syncing colors, so only transform dictionary.tokens.color
+      const transformedTokens = transform(dictionary.tokens.color, { defaultTokenset: false, cleanMeta: ['filePath', 'isSource', 'original', 'attributes', 'category', 'type', 'path'] });
+      return JSON.stringify(transformedTokens, null, 2);
+    },
+  },
   platforms: {
+    figma: {
+      transformGroup: 'js',
+      buildPath: 'dist/tokens/figma/',
```

Comment:
Since this is only generating colors, I wonder if we should create a specific folder for it (in case eventually Figma supports other token types too - fonts & spacing) 🤔 

So like: 

```
    figma-colors: {
      transformGroup: 'js',
      buildPath: 'dist/tokens/figma/colors',
```

Reply:
I like this idea. Even if Figma never supports things like fonts and spacing, I feel like having `colors` added to the buidpath provides some immediate clarity for anyone looking at project structure etc.